packages/matter-node.js/README.md

```diff
@@ -149,6 +149,7 @@ The following parameters are available:
 * -ip: the IP address of the device to commission
 * -discriminator: the discriminator to use for pairing (default: 3840)
 * -pin: the pin to use for pairing (default: 20202021)
+* -code: code to use for pairing (-discriminator and -pin will be ignored)
```

Comment:
Thats the same as "passcode" on DeviceNode right? SHould we then also name it that way so that it is easier mappable?

Reply:
Hehe no it is not the same ... but maybe call it better "pairingcode" like also in other places, ok?

Reply:
> Could you please also add a test for decoding? 

Yeah, it occurred to me after i submitted this that a test would have probably been good.   will add one shortly, thanks!